examples/strong-docs/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,37 +0,0 @@
-<!--
-Questions:
-  https://groups.google.com/forum/#!forum/loopbackjs
-  https://gitter.im/strongloop/loopback
-Immediate support:
-  https://strongloop.com/api-connect-faqs/
-  https://strongloop.com/node-js/subscription-plans/
--->
-
-# Description/Steps to reproduce
```

Comment:
Huh, I am confused - did you accidentally import `strong-docs` into `examples` directory? It looks like you are moving the imported source code from `examples/strong-docs` to `packages/strong-docs`. 

The pull request diff is humongous and very difficult to review as a result :(

I am also concerned about making code archaeology very difficult in the future.

Can you please re-run the import again (or simply copy the code from strong-docs over as I suggested in my comment above) and ensure the files are added to `packages/{new package name}` from the beginning?